test/e2e/test_cli.py

```diff
@@ -39,8 +39,8 @@ def tearDownClass(cls) -> None:
 
     def test_validate_lists_section_with_schema(self) -> None:
         """Test if the resource file 'src/dsp_tools/resources/schema/lists-only.json' can be accessed."""
-        cmd = f"dsp-tools create --lists-only --validate-only {self.test_project_systematic_file.absolute()}"
-        self._make_cli_call(cmd)
+        args = ["create", "--lists-only", "--validate-only", str(self.test_project_systematic_file.absolute())]
+        self._make_cli_call(args)
```

Comment:
personally, I find the `"...".split()` more readable than the list

Reply:
Actually, having now seen how it's used below, I would leave the string in here entirely, and only split it in `_make_cli_call()`

Reply:
Unfortunately, both of your suggestions are not possible, because if the path contains a whitespace, this will be splitted, too.